tests/onnx/models.py

```diff
@@ -983,21 +981,37 @@ def __init__(self):
             name=tensor_name, tensor_array=tensor, data_type=onnx.TensorProto.FLOAT
         )
 
-        mul_node = onnx.helper.make_node(
-            name=model_mul_op_name,
-            op_type="Mul",
-            inputs=[model_cast_output, tensor_name],
-            outputs=[model_output_name],
+        conv_output_node_name = "Conv1_Y"
+        conv_in_channels, conv_out_channels, conv1_kernel_shape = 3, 32, (3, 3)
+        rng = get_random_generator()
+        conv_W = rng.uniform(0, 1, (conv_out_channels, conv_in_channels, *conv1_kernel_shape)).astype(np.float32)
+        conv_B = rng.uniform(0, 1, conv_out_channels).astype(np.float32)
+
+        conv_W_initializer_tensor_name = "Conv1_W"
+        conv_W_initializer_tensor = create_initializer_tensor(
+            name=conv_W_initializer_tensor_name, tensor_array=conv_W, data_type=onnx.TensorProto.FLOAT
+        )
+        conv_B_initializer_tensor_name = "Conv1_B"
+        conv_B_initializer_tensor = create_initializer_tensor(
+            name=conv_B_initializer_tensor_name, tensor_array=conv_B, data_type=onnx.TensorProto.FLOAT
         )
 
-        Y = onnx.helper.make_tensor_value_info(model_output_name, onnx.TensorProto.FLOAT, input_shape)
+        conv_node = onnx.helper.make_node(
+            name="Conv1",
+            op_type="Conv",
+            inputs=[model_cast_output, conv_W_initializer_tensor_name, conv_B_initializer_tensor_name],
+            outputs=[conv_output_node_name],
+            kernel_shape=conv1_kernel_shape,
+        )
+
+        Y = onnx.helper.make_tensor_value_info(conv_output_node_name, onnx.TensorProto.FLOAT, input_shape)
 
         graph_def = onnx.helper.make_graph(
-            nodes=[reciprocal_node, cast_node, mul_node],
+            nodes=[reciprocal_node, cast_node, conv_node],
```

Comment:
This model was used to test the `float64` input to be quantized. Before, when FQ elementwise was not removed it was checked properly. But at this PR this quantizer from Mul is removed. and the test purpose of the model was lost. So, to make it meaningful I changed Mul to Conv.

Reply:
Great, thanks for the explanation.
Interesting that this model was updated to keep the quantizer anyway, but model `tests/torch/data/reference_graphs/quantized/synthetic_model/min.dot` was not.
Update all models to keep the original intention or do not update anything at all, please. Otherwise this is not consistent way.

Reply:
done, please, check